lib/tasks/migrations/create_ivl_benefit_packages_DC.rake

```diff
@@ -14,9 +14,12 @@ namespace :import do
 
     year = ENV['year'].to_i
 
+    slcsp_hios_id = "94506DC0390006-01"
+    slcsp_hios_id = "94506DC0390005-01" if year == 2024
+
     # Second lowest cost silver plan
-    slcs_products = BenefitMarkets::Products::Product.where(hios_id: "94506DC0390006-01")
-    current_slcsp =  slcs_products.select{|a| a.active_year == year}.first
+    slcs_products = BenefitMarkets::Products::Product.where(hios_id: slcsp_hios_id).by_year(year)
```

Comment:
@raghuramg Can we send this information as an argument and default the value to old/new slcsp_hios_id just like the year?

Reply:
done